.github/workflows/build-fedora.yml

```diff
@@ -0,0 +1,64 @@
+# This workflow builds the AYAB package for Ubuntu/Debian Linux
+# and stores it as an artifact
+
+name: build-fedora
+
+on:
+  push:
+    branches:
+      - master
+      # - '*-dev'
+
+jobs:
+  build:
+    runs-on: ubuntu-16.04
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+        fetch-depth: 0
+    - name: Set GIT_TAG variable
+      run: echo "::set-env name=GIT_TAG::$(git describe --tags)"
+    - name: Set PACKAGE_VERSION
+      run: |
+        echo ${{env.GIT_TAG}} > src/main/resources/base/ayab/package_version
+        sed -i 's/PACKAGE_VERSION/${{env.GIT_TAG}}/' src/build/settings/base.json
+        cat src/main/resources/base/ayab/package_version
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6
```

Comment:
Perhaps make also use of `strategy: matrix: python-version...`? -> ${{ matrix.python-version }}

Reply:
OK